birdhouse/.gitignore

```diff
@@ -1 +1 @@
-./config
+config/
```

Comment:
This will match all `config` directories recursively which is *not* what we want. 

From the [gitignore documentation](https://git-scm.com/docs/gitignore#_pattern_format):

> If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.

I think we need to change it to:

```
/config/
```

instead

Reply:
You are absolutely right !   Thanks for catching this enormous error !!!  Fixed.